### PR TITLE
Portfolo contact form mobileversion

### PR DESCRIPTION
**Pull request by siddghosh108**
Added the contact form section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -251,8 +251,59 @@ <h4 class="langs-text">CSS</h4>
...
+      cols="10"
+      rows="5" aria-label="Enter message"
+      ></textarea>
+          <div class="btn-center">
+            <button type="submit" class="form-btn">Get In Touch</button>
+          </div>
+        </form>
+      </div>
+    </div>
+  </footer>
```
Comment (levy002) [inline index.html:307]: - [ ] Kindly make sure to add the background image in order to match the provided Figma design as much as possible.

  - ### YOUR DESIGN
  
   image

  - ### FIGMA DESIGN
  
   image

File: index.html [hunk 2]
```diff
@@ -251,8 +251,59 @@ <h4 class="langs-text">CSS</h4>
...
+      id="user_message"
+      placeholder="Write your message here"
+      required
+      maxlength="500"
+      cols="10"
+      rows="5" aria-label="Enter message"
+      ></textarea>
+          <div class="btn-center">
+            <button type="submit" class="form-btn">Get In Touch</button>
+          </div>
```
Comment (levy002) [inline index.html:303]: - [ ] Kindly make sure to align the `Get in Touch`  button as it is on the provided Figma design as much as possible.

   - ### FIGMA DESIGN
   
    image

   - ### YOUR DESIGN
   
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @siddghosh108   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — COMMENTED** [review]
# INVALID ❌ 

Hi @siddghosh108 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😓

## Invalid Code Review Request ❌
- [ ] Unfortunately, you have submitted a pull request with failing linters. Please make sure that all linters are passing and a tick mark ✔ is present.

_**Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready**_ 🤝🙏

Cheers and Happy coding!👏👨‍💻💻

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by osamaashraf6 — APPROVED** [review]
Hi @siddghosh108 👋, 
# STATUS: APPROVED! 🥇🟢🟢💯 


Your project is complete!  There is nothing else to say other than... it's time to merge it : ship it: 
Congratulations! 🎉 

## Highlights💯🎈🪄✔️


- [x] No linter errors. ✔️
- [x] Good usage in correcting GitHub flow.✔️
- [x] Nice design.✔️


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — COMMENTED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Project Already approved✔️ 


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**